index.js

```diff
@@ -212,7 +214,17 @@ function tsLookup({dependency, filename, tsConfig}) {
     options.module = ts.ModuleKind.AMD;
   }
 
-  const host = ts.createCompilerHost({});
+  var host = ts.createCompilerHost({});
```

Comment:
Is there no way to provide the overrides below as options to the creation of the compiler host? Monkeypatching the methods are more prone to breakages during version bumps if the host lib isn't aware of the need for customization.

Reply:
I can't see any way of configuring this behaviour for the default compilerHost using the options.
The choice seems to be, either monkey patch the methods (which are part of the ts.CompilerHost interface so should be pretty stable) or to implement your own custom ts.CompilerHost.

I have implemented a custom ts.CompilerHost before, over here for example: https://github.com/dazinator/NetPack-Typescript/blob/develop/src/compilerhost.ts

We could do either, I went for the compromise based on the fact the api's that are monkey patched should be part of the ts.CompilerHost contract / interface that all hosts implement.

If you would prefer a custom host, its a bit more work, and a bit more to maintain moving forwards, but we could do that as well.

Reply:
Similar to https://github.com/dependents/node-filing-cabinet/pull/68#discussion_r241990149, the overrides are fine. Thanks.